documentation/source/CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to InferTrade
```

Comment:
Suggestion, substitute the contents of both `documentation/source/CONTRIBUTING.md` and `documentation/source/README.md` for simple includes, as indicated here (in the first code block):  https://myst-parser.readthedocs.io/en/latest/using/howto.html#include-a-file-from-outside-the-docs-folder-like-readme-md

This way, we could avoid having to keep these documents synced up with the ones that exist at the root of the repository.

Reply:
Yes, exactly - we shouldn't be duplicating the content but rather including by reference.

Reply:
@fhashim , I've also commited commit 83fa5d91545154779cfbb12602fc212bc31c98a1 to address this suggestions.